src/renderer/components/deposit/withdraw/Withdraw.tsx

```diff
@@ -325,15 +333,15 @@ export const Withdraw: React.FC<Props> = ({
         <Styled.AssetLabel asset={AssetRuneNative} />
         <Styled.OutputLabel weight={'bold'}>
           {formatAssetAmountCurrency({
-            amount: baseToAsset(runeAmountToWithdraw),
+            amount: getTwoSigfigAssetAmount(baseToAsset(runeAmountToWithdraw)),
             asset: AssetRuneNative,
             decimal: THORCHAIN_DECIMAL,
             trimZeros: true
           })}
           {/* show pricing if price asset is different only */}
           {!eqAsset.equals(AssetRuneNative, selectedPriceAsset) &&
             ` (${formatAssetAmountCurrency({
-              amount: baseToAsset(baseAmount(runeAmountToWithdraw.amount().times(runePrice))),
+              amount: getTwoSigfigAssetAmount(baseToAsset(baseAmount(runeAmountToWithdraw.amount().times(runePrice)))),
```

Comment:
Pass THORCHAIN_DECIMAL to `baseAmount` as second parameter. No matter what default decimal of `baseAmount` is - it will always help to read/understand the code, as well.